src/test/java/model/HeightTest.java

```diff
@@ -0,0 +1,30 @@
+package model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.*;
+
+class HeightTest {
+
+    @Test
+    @DisplayName("Height 객체의 높이가 유효한 값일 때 정상적으로 생성되는지 검증한다.")
+    void shouldCreateHeightWithValidValue() {
+        Height height = new Height(5);
+        assertThat(height.getValue()).isEqualTo(5);
+    }
+
+    @Test
+    @DisplayName("최소값일 때 Height 객체가 정상적으로 생성되는지 검증한다.")
+    void shouldCreateHeightWithMinimumValue() {
+        Height height = new Height(2);
+        assertThat(height.getValue()).isEqualTo(2);
+    }
+
+    @Test
+    @DisplayName("Height 객체의 높이가 2 미만일 때 예외가 발생하는지 검증한다.")
+    void shouldThrowExceptionForHeightBelowMin() {
+        assertThatThrownBy(() -> new Height(1))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("사다리 높이는 2 이상이여야 합니다.");
+    }

```

Comment:
```suggestion
    @Test
    @DisplayName("높이가 2 미만일 때 예외가 발생한다.")
    void shouldThrowExceptionForHeightBelowMin() {
        assertThatThrownBy(() -> new Height(1))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessage("사다리 높이는 2 이상이여야 합니다.");
    }
```

클래스명이 바뀌더라도 테스트의 `@DisplayName`이 바뀌지 않으며,
예외가 발생하는 것을 테스트하기 때문에, 예외가 발생한다. 까지만 작성해도 충분할 것 같은데
어떻게 생각하시나요?

Reply:
네! 제가 테스트 `@DisplayName` 컨벤션을  `-를 검증한다`로 통일해서 작성하였는데,
같은 의미의 반복이라면 말씀해주신대로 더 간결하게 작성해도 충분할 것 같네요
리팩토링 완료했습니다. 감사합니다!